test/e2e-tests.sh

```diff
@@ -54,14 +62,20 @@ function knative_teardown() {
 
 # Setup resources common to all eventing tests
 function test_setup() {
+  # Create the test namespace
+  echo ">> Creating namespace $E2E_TEST_NAMESPACE"
   kubectl create namespace ${E2E_TEST_NAMESPACE} || return 1
+
+  echo "Label the namespace to create the default Broker"
+  kubectl label namespace ${E2E_TEST_NAMESPACE} knative-eventing-injection=enabled
+
   # Publish test images
   $(dirname $0)/upload-test-images.sh e2e || fail_test "Error uploading test images"
 }
 
 function test_teardown() {
   # Delete the test namespace
-  echo "Deleting namespace $E2E_TEST_NAMESPACE"
+  echo ">> Deleting namespace $E2E_TEST_NAMESPACE"
```

Comment:
What is the significance of '>>'?

Reply:
It does not have much significance, just make it clearer in the logs.